extensions/tinypng/package.json

```diff
@@ -56,6 +56,14 @@
       "type": "password",
       "required": true
     },
+    {
+      "title": "Destination Folder",
+      "name": "destinationFolder",
+      "description": "Set Destination Folder.",
```

Comment:
This could maybe be explained better, otherwise people are adding ~/paths or absolute paths?

Reply:
Thank you for your review. I’ll fix it.

Reply:
@pernielsentikaer 

I've revised the description and implementation so that you can now specify either a relative or absolute path for the destination folder.

https://github.com/raycast/extensions/pull/6102/commits/d1f264a905d59de46394a71c78e7dbfcb86b47a3